pkg/graph/schema.graphql

```diff
@@ -208,49 +208,68 @@ type CedarThreat {
 BusinessOwnerInformation contains information about the business owner for a CEDAR system
 """
 type CedarBusinessOwnerInformation {
-	beneficiaryAddressPurpose: [String!]!
-	beneficiaryAddressPurposeOther: String
-	beneficiaryAddressSource: [String!]!
-	beneficiaryAddressSourceOther: String
-	costPerYear: String
-	isCmsOwned: Boolean
-	numberOfContractorFte: String
-	numberOfFederalFte: String
-	numberOfSupportedUsersPerMonth: String
-	storesBankingData: Boolean
-	storesBeneficiaryAddress: Boolean
+  beneficiaryAddressPurpose: [String!]!
+  beneficiaryAddressPurposeOther: String
+  beneficiaryAddressSource: [String!]!
+  beneficiaryAddressSourceOther: String
+  beneficiaryInformation: [String!]!
```

Comment:
Can you doc (either through the mock data or comments) what kind of data we expect to come back through here?

Reply:
I do not know exactly what we can expect here - this is all we are given via API docs:

```go

	// beneficiary information
	BeneficiaryInformation []string `json:"beneficiaryInformation"`
```

Reply:
updated mock data - thanks Clay! Found some info in the data dictionary